ksql-streams/src/main/java/io/confluent/ksql/execution/streams/materialization/ks/KsLocator.java

```diff
@@ -81,18 +87,17 @@
     final Set<HostInfo> standByHosts = metadata.getStandbyHosts();
     LOG.debug("Before filtering: Active host {} , standby hosts {}", activeHost, standByHosts);
 
-    final Stream<KsqlHostInfo> active = Stream.of(asKsqlHost(activeHost));
-    final Stream<KsqlHostInfo> standby = standByHosts
-        .stream()
-        .map(this::asKsqlHost);
-    final Stream<KsqlHostInfo> hostStream = Stream.concat(active, standby);
+    final List<KsqlHostInfo> allHosts = Stream.concat(Stream.of(activeHost), standByHosts.stream())
```

Comment:
Even without a ranking interface.. given we place the activeHost first.. I think we are guaranteed to send to active if its alive and not restoring (i.e laggy)

Reply:
Yep, I think that's true.  Given our discussion earlier about always ranking active first, even during restore, ranking is solely for the benefit of multiple standbys.  Maybe this is low value given that there are likely to be 1 or at most 2 standbys in practice.  What do you think?